README.md

```diff
@@ -1 +1,38 @@
-Home Work #1 - Clean Code
```

Comment:
Very good work and code style. Code has good structure and organization. Very informative readme file. Took some useful hints for my future work)))